cmd/podman/manifest/remove.go

```diff
@@ -10,9 +10,9 @@ import (
 
 var (
 	removeCmd = &cobra.Command{
-		Use:               "remove LIST IMAGE",
-		Short:             "Remove an entry from a manifest list or image index",
-		Long:              "Removes an image from a manifest list or image index.",
+		Use:               "remove LIST DIGEST",
+		Short:             "Remove an item from a manifest list or image index",
+		Long:              "Removes an item from a manifest list or image index.",
 		RunE:              remove,
 		Args:              cobra.ExactArgs(2),
 		ValidArgsFunction: common.AutocompleteImages,
```

Comment:
If a image is not correct then please change the shell completion for images here, I guess we still want the first arg the images to be completed? (is there a way to only list manifest lists for the first arg and no all images? and then if possible provide the autocompletion for the digest as second arg?)